src/ray/core_worker/reference_count.cc

```diff
@@ -1239,9 +1229,6 @@ bool ReferenceCounter::HandleObjectSpilled(const ObjectID &object_id,
     if (!spilled_node_id.IsNil()) {
       it->second.spilled_node_id = spilled_node_id;
     }
-    if (size > 0) {
-      it->second.object_size = size;
```

Comment:
We delete this because we always know the object size ?

Reply:
Yea, we have other channel to know the object size. Also for Add/Remove updates, we don't send the object size.